tests/conftest.py

```diff
@@ -17,7 +17,7 @@ def sql_with_ctes() -> str:
         ccc as (select 3 as cc, aa from aaa),
         ddd as (select 4 as dd, aa from aaa where aa not in (select bb from bbb))
 
-        select *
+        select *, $something
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Missing test case for parameterized SQL handling.

Given the PR's goal to support parameterized SQL, it's crucial to add a test case that verifies the system correctly handles SQL queries with parameters like `$something`. This ensures the new functionality works as expected and prevents regressions in the future.

```suggestion
        select *, :something
        from ccc
            inner join ddd using (aa)
```

Reply:
Is this comment helpful?

Reply:
Is the comment type correct?

Reply:
Is the comment area correct?